postcode_lookup.R

```diff
@@ -0,0 +1,149 @@
+#####################################################
+# Postcode Lookup
+# Author: Catherine Holland
+# Date: February 2022
+# Written on RStudio Server
+# Version of R - 3.6.1
+# Input -
+# Description -
+#####################################################
+
+library(dplyr)
+library(tidyr)
+library(haven)
+
+
+## read data in ##
+
+# postcode data
+pc_file <- read_spd_file()
+
+# simd data
+read_simd_file <- function(file) {
+  simd_path <- fs::path(get_lookups_dir(), "Deprivation", file)
+  # If given a sav extension (or other), swap it for rds
+  simd_path <- fs::path_ext_set(simd_path, "rds")
+  # Check if the file exists and we can read it
+  if (!fs::file_access(simd_path, "read")) {
+    rlang::abort(message = "Couldn't read the simd file")
+  }
+  return(simd_path)
+}
```

Comment:
Move this function to its own script, please. I think called `get_simd_path`, since it actually returns the path, it doesn't read the file. I think the SPD function is badly named but we can deal with that separately!